archive.php

```diff
@@ -0,0 +1,33 @@
+<?php
+	include 'navbar.php';
+?>
+
+<html>
+<head>
+<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="archive.css">
+</head>
+<body>
+	<div id="container">
+		<script>
+		var youtube_videos = ['https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
+								,'https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
+							];
+		for(i=0;i<youtube_videos.length;i++){
+			$("#container").append("<div class='col-lg-4 col-md-4 col-sm-6 col-sm-12' id='vid'><iframe frameborder='0' allowfullscreen></iframe></div>");
+			
+		}
+			var video = $('#vid');
+			
+		for(i=0;i<youtube_videos.length;i++){
+			
+			video.children('iframe').attr('src',youtube_videos[i]);
+			video = video.next();
+		}
+		</script>
+	</div>
+<script src="arcive.js"></script>	
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+</body>
+</html>
```

Comment:
How do i add new line should i add in the script tags or should i just add
a <br>

On Sat, Aug 29, 2015 at 8:25 PM, Siddharth Kannan <notifications@github.com>
wrote:

> In archive.php
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38268154>
> :
>
> > +			
> > +		}
> > +			var video = $('#vid');
> > +			
> > +		for(i=0;i<youtube_videos.length;i++){
> > +			
> > +			video.children('iframe').attr('src',youtube_videos[i]);
> > +			video = video.next();
> > +		}
> > +		</script>
> > +	</div>
> > +<script src="arcive.js"></script>	
> > +<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
> > +<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
> > +</body>
> > +</html>
>
> Add a new line at the end of this file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38268154>
> .
>